lib/Star/Kata/Command/StartCommand.php

```diff
@@ -60,18 +61,11 @@ public function configure()
      */
     protected function execute(InputInterface $input, OutputInterface $output)
     {
-        try {
-            $kataName = $input->getArgument('kata');
-            if (empty($kataName)) {
-                throw new \RuntimeException('Kata must be supplied.');
-            }
+        $kataName = $input->getArgument('kata');
 
-            $kata = $this->collection->getKata($kataName);
-            if (null === $kata) {
-                throw new \RuntimeException("The '{$kataName}' kata was not found.");
-            }
+        try {
+            $kata = $this->service->startKata($kataName);
 
-            $kata->start();
             $output->writeln('<info>' . $kata->getDescription() . '</info>');
         } catch (Exception $e) {
```

Comment:
you shouldn't catch all `\Exceptions` just the specific type you know calling `stateKata`'s `interface` may throw.